apis/Google.Cloud.DevTools.Common/Google.Cloud.DevTools.Common/project.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": "1.0.0-alpha00-*",
+  "description": "Common Protocol Buffer messages for Google Cloud Developer Tools APIs.",
+
+  "packOptions": {
+    "summary": "Google Cloud Developer Tools APIs Common Protos",
+    "description": "Common Protocol Buffer messages for Google Cloud Developer Tools APIs.",
+    "owners": [ "google-cloud" ],
+    "tags": [ "Google", "Cloud", "Tools" ],
+    "iconUrl": "https://cloud.google.com/images/gcp-icon-64x64.png",
+    "projectUrl": "https://github.com/GoogleCloudPlatform/google-cloud-dotnet",
+    "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0",
+    "requireLicenseAcceptance": false
+  },
+
+  "buildOptions": {
+    "keyFile": "../../GoogleApis.snk",
+    "xmlDoc": true
+  },
+  "dependencies": {
+    "Google.Api.CommonProtos": "1.0.0",
+    "Google.Protobuf": "3.2.0",
+    "Google.Protobuf.Tools": {
+      "version": "3.2.0",
+      "type": "build"
+    }
+  },
+  "frameworks": {
+    "net45": {},
+    "netstandard1.3": {}
```

Comment:
Yes, because gRPC needs that. This is just protos, at least at the moment. We certainly *could* make it 1.5, but we don't need to. If we ever decided to consume this in something using the REST API, it would be useful to have 1.3, potentially...